arch/csr/Smcntrpmf/minstretcfg.yaml

```diff
@@ -1,74 +1,68 @@
----
 # yaml-language-server: $schema=../../../schemas/csr_schema.json
 $schema: csr_schema.json#
 kind: csr
 name: minstretcfg
-long_name: Machine Counter Configuration for Instret
+long_name: Machine Instructions-Retired Counter Configuration
 address: 0x322
 priv_mode: M
 length: 64
+base: 64
 definedBy: Smcntrpmf
 description: |
   The `minstretcfg` CSR is a 64-bit machine-level register that configures privilege
-  mode filtering for the `minstret` (retired instruction) counter. Each inhibit bit (xINH)
+  mode filtering for the `minstret` (Machine Instructions-Retired Counter). Each inhibit bit (xINH)
   disables counting of retired instructions in the associated privilege mode.
 
   Bits 62-58 correspond to the following privilege modes:
-    - MINH: M-mode
-    - SINH: S/HS-mode
-    - UINH: U-mode
-    - VSINH: VS-mode
-    - VUINH: VU-mode
+    - MINH: If set, then counting of events in M-mode is inhibited.
+    - SINH: If set, then counting of events in S/HS-mode is inhibited.
+    - UINH: If set, then counting of events in U-mode is inhibited.
+    - VSINH: If set, then counting of events in VS-mode is inhibited.
+    - VUINH: If set, then counting of events in VU-mode is inhibited.
```

Comment:
This is fine if you don't want to implement a table, but let's make sure both are the same format. Either way, but pick one way. ;-)